src/migrations/hotbar-store/5.0.0-alpha.2.ts

```diff
@@ -0,0 +1,19 @@
+// Cleans up a store that had the state related data stored
+import { Hotbar } from "../../common/hotbar-store";
+import { migration } from "../migration-wrapper";
+import { v4 as uuid } from "uuid";
+
+export default migration({
+  version: "5.0.0-alpha.2",
+  run(store) {
+    const hotbars = (store.get("hotbars") || []) as Hotbar[];
+
+    hotbars.forEach((hotbar) => {
+      if (!hotbar.id) {
+        hotbar.id = uuid();
+      }
+    });
+
+    store.set("hotbars", hotbars);
```

Comment:
nit: I find assigning to the input confusing when reading code.
```suggestion
    const rawHotbars = (store.get("hotbars") || []) as Hotbar[];

    store.set("hotbars", rawHotbars.map(({ id = uuid.v4(), ...rest }) => ({
      id,
      ...rest,
    }));
```

Reply:
Refactored a bit, your suggestion was a bit confusing to my brain so the code is now somewhere in the middle.

Reply:
okay